app/services/import/persist-licence.service.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+/**
+ * Persists the licence
+ * @module PersistLicenceService
+ */
+
+const LicenceModel = require('../../models/licence.model.js')
+const RegionModel = require('../../models/region.model.js')
+
+async function go (licence) {
+  const {
+    licence: {
+      regionCode,
```

Comment:
put this straight in if no diff with the mapper. 

region might be issue and other things on the licence might not work